src/main/kotlin/co/netsoc/netsocbot/commands/CommandFunctions.kt

```diff
@@ -27,4 +27,46 @@ suspend fun register(message: Message): Unit {
         }
         messageUser(author, "Please message me your UCC email address so I can verify you as a member of UCC")
     }
+}
+
+fun brownie(message: Message): String {
+    val receiver = message.usersMentioned.firstOrNull()
+    val author = message.author
+
+//    gonna use a file cause i dont know if im allowed to use a databse
+    var leaderboardFile = File("brownieLeaderboard.txt")
+    var yes = leaderboardFile.readLines()
+    if (receiver != null && author.id != receiver.id)  {
```

Comment:
I would add a check to see that the author has not given someone else a brownie already today, could get quite spammy as it is now. This could be checked by simply storing author ID to the timestamp of the last time they gave a brownie point.